lib/Sema/Constraint.cpp

```diff
@@ -337,22 +337,37 @@ void Constraint::print(llvm::raw_ostream &Out, SourceManager *sm) const {
     if (isIsolated())
       Out << " (isolated)";
 
-    if (Locator) {
+    auto firstLocator = Locator;
```

Comment:
Sorry one more small change - let’s use Locator directly instead of just re-assigning it into a variable.